src/main/java/frc/robot/Robot.java

```diff
@@ -43,9 +45,10 @@
   public Robot()
   {
     // Instantiate Subsystems
+    sensors = new Sensors();
     climber = new Climber();
     conveyor = new Conveyor();
-    drivetrain = new Drivetrain();
+    drivetrain = new Drivetrain(sensors);
```

Comment:
Generally you don't wanna pass in one subsystem to another.  Subsystems are abstractions of hardware on the robot and should be able to operate independently.  If drivetrain and sensors need to be used in conjunction, you would do that in a command.  Otherwise, you can always put the sensors in drivetrain and override `periodic()`.